test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -396,3 +396,40 @@ Describe "New-Item -Force should throw an error for invalid characters in direct
         { New-Item -Path $invalidPath -ItemType Directory -Force -ErrorAction Stop } | Should -Throw -ErrorId 'CreateDirectoryIOError,Microsoft.PowerShell.Commands.NewItemCommand'
     }
 }
+
+Describe "If the target path contains wildcard characters, the operation should succeed in creating a file link." -Tags @("CI", 'RequireAdminOnWindows') {
+    BeforeAll {
+        $testFolder     = '[LinkCreation]'
+        $testFile       = '[test].txt'
+
+        $symbolicLink   = 'sym-lk.txt'
+        $hardLink       = 'hard-lk.txt'
+        $junction       = 'junc'
+        $fileInJunction = Join-Path $junction $testFile
+        
+        $expectedFileContent = 'succeed!'
+
+        New-Item "TestDrive:\$testFolder" -Type Directory
+        Push-Location -PSPath "TestDrive:\$testFolder"
```

Comment:
Please don't use alias PSPath. We should explicitly use `Path` or `LiteralPath`.